train.py

```diff
@@ -27,11 +28,14 @@ def main(args):
 
     envs, datasets = load_envs(args.datasets) # Load Minari datasets and corresponding Gym environments
 
+    # load the text dataset
+    text_dataset = load_dataset('wikitext', 'wikitext-103-raw-v1')
```

Comment:
I also think we want to initial text datasets in a similar manner, where each dataset is wrapped in a class like in Task, which contains functions for sampling from the task, and evaluating, etc.

Reply:
Agree, will do so in future commit.